include/fastq_conversion.hpp

```diff
@@ -0,0 +1,14 @@
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/io/sequence_file/all.hpp>
+#include <seqan3/io/sequence_file/input.hpp>
+#include <seqan3/io/sequence_file/output.hpp>
+
+void convert_fastq(std::filesystem::path fastq_file, std::filesystem::path out)
+{
+    seqan3::sequence_file_input fin{fastq_file};
+    seqan3::sequence_file_output fout{std::cout, seqan3::format_fasta{}};
```

Comment:
this should be in an `else` branch, no?

Reply:
Why? Either I don't get an output then it is empty or I get when then I use the if. Don't see a reason to put it in an else